pkg/sql/logictest/testdata/logic_test/create_statements

```diff
@@ -1042,6 +1042,23 @@ CREATE TABLE crdb_internal.session_variables (
    value STRING NOT NULL,
    hidden BOOL NOT NULL
 )  {}  {}
+CREATE TABLE crdb_internal.statement_statistics (
+   aggregated_ts TIMESTAMPTZ NOT NULL,
+   fingerprint_id BYTES NOT NULL,
+   plan_hash INT8 NOT NULL,
+   app_name STRING NOT NULL,
+   metadata JSONB NOT NULL,
+   statistics JSONB NOT NULL,
+   plan JSONB NOT NULL
+)  CREATE TABLE crdb_internal.statement_statistics (
+   aggregated_ts TIMESTAMPTZ NOT NULL,
+   fingerprint_id BYTES NOT NULL,
+   plan_hash INT8 NOT NULL,
+   app_name STRING NOT NULL,
+   metadata JSONB NOT NULL,
```

Comment:
In the future, we may want to consider a view which would normalize the JSONB COLUMNS (statistics and metadata) so users wouldn't have to do this on their own. 

It would make it easier to consume as well for those who rely on the DDL to understand what fields are available to them. Currently, you need to query the JSONB columns to know what's available.

Reply:
True. I think this is another point that suggest us to use switch this to a internal virtual table and exposes a user friendly virtual view on top of it.